cdap-docs/examples-manual/source/examples/data-cleansing.rst

```diff
@@ -30,7 +30,7 @@ of the application are tied together by the class ``DataCleansing``:
 
 .. literalinclude:: /../../../cdap-examples/DataCleansing/src/main/java/co/cask/cdap/examples/datacleansing/DataCleansing.java
    :language: java
-   :lines: 32-
+   :lines: 30-
```

Comment:
Well, until that change comes back, and the two are sync'd the docs will show the code incorrectly. If you are bring the change back, then in that PR you must:
- change the Java file
- change the Doc file
- change the MD5 hash.

It's best if these are done together...